src/styles/blocks/product.scss

```diff
@@ -0,0 +1,14 @@
+.product {
+  max-width: 600px;
+  &__photo {
+    width: 100%;
+    margin-bottom: 24px;
+
+    @include imageTransform;
+  }
+
+  &__price {
+    font-family: "Space Grotesk", sans-serif;
```

Comment:
Since most of the page uses "Space Mono" i wrote it as global font family. I use "Space Grotesk" only for titles and lines that needs an accent. So maybe i can leave as it is?

Reply:
Good point!